### DISCUSSION TITLE
save_report formatting not working

### DISCUSSION THREAD

**Discussion by paulhighams** [datapane/datapane#396] [Help] [answered]
### Summary

save_report works until I add a formatting section then it errors, saying datapane has no attribute 'Formatting'.

I have used the exact formatting section given in the datapane documentation

### Additional information

```js
formatting=dp.Formatting(light_prose=False,
        	accent_color="orange",
        	bg_color="#EEE",
        	text_alignment=dp.TextAlignment.LEFT,
        	font=dp.FontChoice.MONOSPACE,
        	width=dp.Width.FULL,)

Error Message:
    formatting=dp.Formatting(light_prose=False,
AttributeError: module 'datapane' has no attribute 'Formatting'
```

**Comment by spookylukey** [comment, marked as answer]
Which datapane version are you using? The docs usually follow the most recent released version, which is 0.16.7 from PyPI - https://pypi.org/project/datapane/

I tried the example on this page: https://docs.datapane.com/reports/styling/ and it worked correctly with that version of datapane.

**Reply by paulhighams** [reply]
I was on 0.16.1, upgrading to 0.16.7 fixed it. Thanks
